docs/guide/persistence_cache.md

```diff
@@ -105,7 +120,11 @@ Redis is used via [Redis pecl extension](https://pecl.php.net/package/redis).
 See [Redis Cache Adapter in Symfony documentation](https://symfony.com/doc/3.4/components/cache/adapters/redis_adapter.html#configure-the-connection)
 for information on how to configure Redis.
 
-Example:
+Out of the box in `app/config/cache_pool/cache.redis.yml` you'll find a default example that can be used.
+
+For eZ Platform Cloud/Platform.sh; This is automatically configured in `app/config/env/platformsh.php` if you have enabled redis as `rediscache` platform.sh service.
+
+For anything else you can enabled it with environment variables detected by `app/config/env/generic.php`; If you set the following environment variables `export CACHE_POOL="cache.redis" CACHE_DSN="secret@example.com:1234/13"`, it will result in config like this:
```

Comment:
```suggestion
For anything else, you can enable it with environment variables detected automatically by `app/config/env/generic.php`. For instance, if you set the following environment variables `export CACHE_POOL="cache.redis" CACHE_DSN="secret@example.com:1234/13" CACHE_NAMESPACE="ez"`, it will result in config like this:
```

Reply:
you don't need to set `CACHE_NAMESPACE="ez"` as it's the default, that is kind of the point here for people to become aware of there being a namespace option also which has default value.

On this one you did not change `;` to `:` btw